src/templates/pt/2019/base.html

```diff
@@ -0,0 +1,229 @@
+{% block methodology_stat_1 %}5.8M{% endblock %}
+{% block methodology_stat_2_title %}Dados processados{% endblock %}
+{% block methodology_stat_2 %}20.9 TB{% endblock %}
+{% block methodology_description %}
+Salvo indicação em contrário, as métricas em todos os 20 capítulos do Web Almanac são originárias do conjunto de dados
+do HTTP Archive. O HTTP Archive é um projeto administrado pela comunidade que acompanha como a Web é criada desde 2010.
+Usando o WebPageTest e o Lighthouse, os metadados de cerca de 6 milhões de sites são testados mensalmente e incluídos em
+um banco de dados público do BigQuery para análise. O conjunto de dados de julho de 2019 foi usado como base para as
+métricas do Web Almanac. Para mais informações, consulte a página Metodologia.
+{% endblock %}
+{% block methodology_link %}Conheça nossa Metodologia{% endblock %}
+
+{% block introduction %}Introdução{% endblock %}
+
+{% block foreword_title %}Prefácio{% endblock %}
+
+{% block foreword %}
+<p>
+  A web aberta é uma rede de tecnologias incrivelmente complexa e em evolução. Indústrias e carreiras inteiras são
+  construídas na Web e dependem de seu vibrante ecossistema para ter sucesso. Por mais crítica que seja a web, entender
+  como está se saindo tem sido surpreendentemente evasivo. Desde 2010, a missão do projeto HTTP Archive é rastrear como
+  a Web é construída e está fazendo um trabalho incrível. No entanto, houve uma lacuna que foi especialmente difícil de
+  resolver: trazer significado aos dados que o projeto HTTP Archive está coletando e permitir que a comunidade entenda
+  facilmente o desempenho da web. É aí que entra o Web Almanac.
+</p>
+
+<p>
+  A missão do Web Almanac é levar o tesouro de insights que, de outra forma, seriam acessíveis apenas a intrépidos
+  mineradores de dados e empacotá-lo de uma maneira fácil de entender. Isso é possível com a ajuda de especialistas do
+  setor, que podem entender os dados e nos dizer o que eles significam. Cada um dos 20 capítulos do Web Almanac se
+  concentra em um aspecto específico da Web, e cada um foi criado e revisado por especialistas em seu campo. A força do
+  Web Almanac flui diretamente da experiência das pessoas que o escrevem.
+</p>
+
+<p>
+  Muitas das conclusões do Web Almanac são dignas de comemoração, mas também é um lembrete importante do trabalho
+  ainda necessário para oferecer experiências de usuário de alta qualidade. As análises orientadas por dados em cada
+  capítulo são uma forma de responsabilidade que todos compartilhamos pelo desenvolvimento de uma Web melhor. Não se
+  trata de envergonhar aqueles que estão errando, mas de iluminar o caminho das melhores práticas, para que haja uma
+  maneira clara e correta de fazer as coisas. Com a ajuda contínua da comunidade da web, esperamos fazer disso uma
+  tradição anual, para que a cada ano possamos acompanhar nosso progresso e fazer as correções necessárias, conforme
+  necessário.
+</p>
```

Comment:
Combine these to keep line numbering same as English:

```suggestion
    <p>
      Muitas das conclusões do Web Almanac são dignas de comemoração, mas também é um lembrete importante do trabalho ainda necessário para oferecer experiências de usuário de alta qualidade. As análises orientadas por dados em cada capítulo são uma forma de responsabilidade que todos compartilhamos pelo desenvolvimento de uma Web melhor. Não se trata de envergonhar aqueles que estão errando, mas de iluminar o caminho das melhores práticas, para que haja uma maneira clara e correta de fazer as coisas. Com a ajuda contínua da comunidade da web, esperamos fazer disso uma tradição anual, para que a cada ano possamos acompanhar nosso progresso e fazer as correções necessárias, conforme necessário.
    </p>
```